ci-operator/step-registry/openshift-extended/web-tests/openshift-extended-web-tests-commands.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+pwd && ls -ltr
+echo $KUBECONFIG
+echo $KUBECONFIG_PATH
+which kubectl
```

Comment:
@yapei it possible cause the step quit with non-zero, so better
`which kubectl || true`

Frankly the tests-private-ui image should no kubectl, so need 
```shell
export HOME=/tmp/home
mkdir -p "${HOME}"
if ! which kubectl; then
    export PATH=$PATH:$HOME
    ln -s "$(which oc)" ${HOME}/kubectl
fi
```

if you want oc, should add `cli: latest` in step definition to inject oc.